styles.css

```diff
@@ -87,4 +88,327 @@
     align-items: center;
     margin: auto;
     gap: 65px;
+}
+.feature-section {
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+    gap: 48px;
+    background: #FFFFFF;
+    width: 1440px;
```

Comment:
As i said before, we can directly add max-width of the page to .app, so we dont need to use this width here.

Reply:
when i add width the content in the feature section align to the left when i zoom out.

Reply:
when i remove the width the content in the feature section aligns to the left when i zoom out.